### PR TITLE
Add crm-public-form-item class to contribution page and add possibili…

### PR DESCRIPTION
**Pull request by guanhuan**
…ty to style profiles consistently

### CODE REVIEW

File: templates/CRM/Contribute/Form/Contribution/Main.tpl [hunk 1]
```diff
@@ -25,8 +25,9 @@
 *}
 {* Callback snippet: On-behalf profile *}
 {if $snippet and !empty($isOnBehalfCallback)}
+<div class="crm-public-form-item crm-section">
   {include file="CRM/Contribute/Form/Contribution/OnBehalfOf.tpl" context="front-end"}
-
+</div>
```
Comment (colemanw) [inline templates/CRM/Contribute/Form/Contribution/Main.tpl]: It looks like you're adding a new closing tag that wasn't there before. Was it missing?

**Comment by colemanw** [comment]
@guanhuan can you take a look at the comment above?

**Comment by guanhuan** [comment]
@colemanw Sorry this got lost in piles of emails. I added a pair of divs to have the new class over them elements so they can be styled as well. But probably not the best approach here. Shall I move the class into OnBehalfOf.tpl instead?

Best

G

**Comment by colemanw** [comment]
Sorry, my fault, I didn't see the opening tag.
Yes, I think moving it into the OnBehalfOf.tpl might be better. You could add the class conditionally based on `$context=='front-end'`.

**Comment by guanhuan** [comment]
No problem. Updated. Please take a look.

**Pull request merged**